S2VX.Game/Story/Command/S2VXCommand.cs

```diff
@@ -56,8 +55,8 @@ public abstract class S2VXCommand : IComparable<S2VXCommand> {
             var systemType = Type.GetType($"S2VX.Game.Story.Command.{commandName}Command");
             var staticMethod = systemType.GetMethod("FromString", BindingFlags.Public | BindingFlags.Static);
             var command = staticMethod.Invoke(null, new object[] { split }) as S2VXCommand;
-            command.StartTime = double.Parse(split[1], CultureInfo.InvariantCulture);
-            command.EndTime = double.Parse(split[2], CultureInfo.InvariantCulture);
+            command.StartTime = S2VXUtils.StringToDouble(split[1]);
```

Comment:
I notice this is the only place that uses Doubles. All commands in practice use Floats.

Reply:
This is intended to match how Drawables handle type values. Time values inside of Drawables are doubles. Position, rotation, scale, etc. are floats.